docs/managed-tests/validation.md

```diff
@@ -0,0 +1,34 @@
+# Validation Tests
+
+Validation tests ensure that your extension's metadata and WooCommerce feature declarations meet baseline requirements. By checking headers, WooCommerce compatibility flags, and theme templates where applicable, these tests help maintain clear, consistent, and accurate information for merchants.
+
+## What Validation Tests Check
+
+- **Plugin Headers:** Ensures that important headers such as `Requires PHP`, `Requires at least`, `Tested up to`, `WC requires at least`, and `WC tested up to` are present and valid. Missing or incorrect headers can lead to confusion or compatibility issues.
+- **WooCommerce Feature Support:** Verifies any declared incompatibilities with WooCommerce features like High Performance Order Storage (HPOS) or Cart and Checkout blocks. Explicit incompatibilities are flagged, encouraging you to support these features where possible.
+- **Theme Templates (For Themes):** Checks if your theme has outdated WooCommerce templates. If found, this results in a warning, guiding you to update templates for the latest WooCommerce standards.
+
+## Possible Outcomes
+
+- **Success:** All required metadata is present and correct, and no invalid headers or incompatible feature declarations were found.
+- **Warning:** Some headers may be missing or incorrect, or you declared incompatibility with features that you should ideally support.
+- **Failed (for Themes):** Outdated templates or critical metadata issues prevent passing the validation.
```

Comment:
This is inaccurate, it's a warning right now.

Reply:
Fixed, check screenshot below